docs/guides/cloud.rst

```diff
@@ -125,8 +152,11 @@ better!
 
 * Please join us on `our Discord <https://discord.gg/umUueND6ag>`_  to ask
   questions.
-* If you're having problems, check `our status page
-  <https://www.edgedbstatus.com/>`_ for information about any service
-  interruptions.
+* If you're experiencing a service interruption, check `our status page
+  <https://www.edgedbstatus.com/>`_ for information on what may be
+  causing it.
 * Report any bugs you find by `submitting a support ticket
-  <https://edgedb.com/p/cloud-support>`_.
+  <https://edgedb.com/p/cloud-support>`_. Note: when using EdgeDB Cloud
+  through the CLI, setting the ``RUST_LOG`` environment variable to ``info``,
+  ``debug``, or ``trace`` may also provide further info on what is happening
+  behind the scenes.
```

Comment:
> may also provide further info on what is happening behind the scenes.

I wonder if this implies we want the user to fix it themselves (when I'm guessing that won't often be possible). Maybe we change it to something like "…may provide additional debugging information which will be useful to include with your ticket."

If the user sees something in the logs they can fix, they'll probably still do it, but this makes it clear we're not trying to put the responsibility onto them.

Reply:
Sounds good, went with your suggestion.